alicptfts/alicptfts.py

```diff
@@ -231,6 +232,45 @@ def generate_headers(self, scan_params, time_start, time_end):
         headers.append(";".join(scan_params))
         return headers
 
+    def calculate_time_one_pass(self, max_range, max_velocity, accel):
+        '''
+        calculates the number of seconds it would take to move from 
+        one end of range to the other end of range
+        '''
+        vf = np.sqrt(accel*max_range)
```

Comment:
you can use math.sqrt. I avoid to use numpy since some old machine does not install it.
and also I suggest that adding type check
`vf = float(np.sqrt(accel*max_range))`

Reply:
fixed